common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -470,6 +470,7 @@ enum class eChannelScanOpErrCode : uint8_t {
 };
 
 #define CHANNEL_SCAN_INVALID_PARAM -1
+#define SCAN_ALL_CHANNELS 0
```

Comment:
I really don't like this (did I say this already?). I still think that splitting APIs is better, but OK, we can do this in a separate PR I guess.